alt1/scripts/notifications.ts

```diff
@@ -54,17 +56,50 @@ export function notifyEvent(event: EventRecord): void {
         return;
     }
 
+    if (event.type === "deleteEvent" && event.id === recentEvent?.id) {
```

Comment:
```suggestion
        if (
        (event.type === "deleteEvent" || (event.type === "editEvent" && event.duration === 0)) &&
        event.id === recentEvent?.id
    ) {
```
Last thing that I can see, otherwise looks good.